llvm/test/CodeGen/Generic/inline-asm-debuginfo.c

```diff
@@ -0,0 +1,17 @@
+// RUN: clang -emit-llvm -c %s -g -o %t
```

Comment:
Due to layering, we cannot use clang in llvm/test. Due to differences in AIX/Windows, it's very difficult to write fully generic test. Just write the IR, possibly generated by Clang https://llvm.org/docs/TestingGuide.html#elaborated-tests